Formula/emojify.rb

```diff
@@ -0,0 +1,15 @@
+class Emojify < Formula
+  desc "Emoji on the command-line :scream:"
+  homepage "https://github.com/mrowa44/emojify"
+  url "https://github.com/mrowa44/emojify/archive/v1.0.0.tar.gz"
+  sha256 "fabefc4767428a2634a77e7845e315725b75b50f282d0943c5b65789650c25d1"
+  head "https://github.com/mrowa44/emojify.git"
+
+  def install
```

Comment:
This seems to be a simple shell script that doesn't need to be built. In this case we don't need to create bottles (binary packages). To denote this, please add `bottle :unneeded` between the `head …` and this line (leave an empty line before and after).